Makefile

```diff
@@ -330,8 +330,9 @@ tarball-clean:
 fetch-testdata:
 	wget -nc -P testdata/btf 'https://github.com/cilium/tetragon-testdata/raw/main/btf/vmlinux-5.4.104+'
 
-.PHONY: generate codegen protoc-gen-go-tetragon
-generate:
+.PHONY: crdgen generate codegen protoc-gen-go-tetragon
+generate: | crds
```

Comment:
genuinely curious, I just discovered that: what's the use for the `|`, I searched and saw it's for "Order-Only Prerequisites" but what's the interest here?

Reply:
I'll be honest I'm not fully certain it's necessary since they're all PHONY targets but I've been burned in the past by not including this. Basically the point is to avoid `make` making (see what I did there) assumptions about the relationship between the two targets, when all I want to do is just run the damn thing.